<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>453</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Mongolian Baiti</family>
    <weight>75</weight>
    <bold>true</bold>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select XML files.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbET">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty Tags</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPC">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>P Class  Tags</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbWS">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Whitespaces</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSC">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Section Content</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSelect">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRun">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Cleanup</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>521</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSingle">
    <property name="geometry">
     <rect>
      <x>517</x>
      <y>10</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Single File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbTable">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRemove">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>MicrosoftTeams-image (8).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>pbSingle</zorder>
   <zorder>label</zorder>
   <zorder>cbET</zorder>
   <zorder>cbPC</zorder>
   <zorder>cbWS</zorder>
   <zorder>cbSC</zorder>
   <zorder>pbSelect</zorder>
   <zorder>pbRun</zorder>
   <zorder>listWidget</zorder>
   <zorder>cbTable</zorder>
   <zorder>pbRemove</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
